src/services/permissions-getter.js

```diff
@@ -19,38 +19,52 @@ class PermissionsGetter {
     return this.configStore.lianaOptions.envSecret;
   }
 
-  cleanCache() {
-    this.permissions = {};
+  cleanCache({ storePermissionsInto } = {}) {
```

Comment:
[MISC] What does `storePermissionsInto` mean. It makes no sense

Reply:
Something like `permissionsCacheKey` or even `environmentId` would be better suited